src/app/features/creature/creature-template-resistance/creature-template-resistance.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+
+import { CreatureHandlerService } from '../creature-handler.service';
+import { MysqlQueryService } from '@keira-shared/services/mysql-query.service';
+import { SingleRowEditorService } from '@keira-abstract/service/editors/single-row-editor.service';
+import {
+  CREATURE_TEMPLATE_RESISTANCE_ID,
+  CREATURE_TEMPLATE_RESISTANCE_TABLE, CREATURE_TEMPLATE_SCHOOL,
+  CreatureTemplateResistance
+} from '@keira-types/creature-template-resistance.type';
+
+@Injectable()
+export class CreatureTemplateResistanceService extends SingleRowEditorService<CreatureTemplateResistance> {
+
+  /* istanbul ignore next */ // because of: https://github.com/gotwarlost/istanbul/issues/690
+  constructor(
+    protected handlerService: CreatureHandlerService,
+    public readonly queryService: MysqlQueryService,
+    protected toastrService: ToastrService,
+  ) {
+    super(
+      CreatureTemplateResistance,
+      CREATURE_TEMPLATE_RESISTANCE_TABLE,
+      CREATURE_TEMPLATE_RESISTANCE_ID,
+      CREATURE_TEMPLATE_SCHOOL,
```

Comment:
?

Reply:
Ah boy do I suck at naming. It should be CREATURE_TEMPLATE_RESISTANCE_SCHOOL, it is a key and I saw in for example creature_template the 2 keys where added here. So I figured it to be the same

Reply:
I am working on it and I already changed that